<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1234567</width>
    <height>123456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Messenger</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="listLayout" stretch="1,13">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="searchLayout" stretch="1,4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="menu_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dialog_search_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>Dialog serach...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="dialogs_list"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="dialog_layout" stretch="1,13">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="headliner" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: white;
border: 0.5px solid grey;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="headliner_layout" stretch="2,5,1">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="dialog_label">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border: none; background: none;</string>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="text">
                <string>Dialog Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="msg_search_edit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius: 15px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Search...</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="chat_buttons_layout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="new_chat_button">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="chat_settings_button">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="P2P_button">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="dialog_window" native="true">
          <property name="styleSheet">
           <string notr="true">border: 0.5px solid grey;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, 
    stop:0 rgba(161, 196, 253, 255), 
    stop:0.5 rgba(194, 233, 251, 255), 
    stop:1 rgba(118, 199, 192, 255));</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
